doc/api/test.md

```diff
@@ -1645,7 +1645,11 @@ changes:
   `fn` does not have a name.
 * `options` {Object} Configuration options for the subtest. The following
   properties are supported:
-  * `concurrency` {number} The number of tests that can be run at the same time.
+  * `concurrency` {number|boolean} If a number is provided,
+    then that many files would run in parallel.
+    If truthy, it would run (number of cpu cores - 1)
+    files in parallel.
+    If falsy, it would only run one file at a time.
```

Comment:
We should use this PR to update this: `1` is truthy, yet it doesn't mean that we would run more than one test at a time. Also, we are no longer using the number of CPU cores.
```suggestion
    If `true`, it would run `os.availableParallelism()  - 1` files in parallel.
    If `false`, it would only run one file at a time.
```

Reply:
Sure, I've updated both `test` and `context.test` with your suggestion as they were both using the same description.